lib/src/flutter_web.dart

```
@@ -52,11 +52,7 @@ $_samplePackageName:lib/
   }
 
   Future<void> warmup() async {
-    try {
-      await initFlutterWeb();
-    } catch (_) {
-      // ignore any exception here
-    }
+    await initFlutterWeb();
```

This try/catch was hiding the flutter setup issues also so I removed it - but can add back in if we need it as a safety check or something? I think it should at least log a warning or severe message though.

---

I really dislike catch and ignore. Happy to try living without it for the time being.

---

We shouldn't propagate any exception from here - warmup is meant to be a best effort, and not stop the initialization of the server. If this call to `initFlutterWeb` fails, a future one (reacting to a user request) could succeed.

I'd catch any exception and log it. Definitely agree that silently catching and ignoring is a good place for bugs to hide.

---

done, logging a warning now